### PR TITLE
scaffolding with Erik Guzman

### PR DESCRIPTION
**Pull request by JJSanch**

### CODE REVIEW

File: spec/requests/recipes_spec.rb [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+      * ½ pound ground beef 1 slice gluten free bread ⅓ cup milk Salt and black pepper to taste 1 egg yolk  ½ cup grated Parmigiano Reggiano cheese.", instructions: 'In a large bowl, combine gluten free bread and milk. Let bread soak, then squeeze out excess milk. In the same bowl, add bread, egg, ground beef, 1/2 the parsley, 1/2 the cheese, and season with salt and pepper. Roll into 3/4 inch balls &amp; set aside. In a large skillet, gently cook the garlic and chili flakes in olive oil. Add meatballs and cook thoroughly. Add the cherry tomatoes and saute for 2 minutes, then season with salt and pepper to taste. Deglaze with wine and reduce mixture by half. Cook the pasta according to package directions, then reserve 1/2 cup of the cooking liquid. Drain the pasta then toss with the sauce and add the reserved cooking liquid. Cook the pasta in the sauce for one minute, remove the skillet from the heat and fold in the remaining cheese and parsley.', ready_in_minutes: 25,cooking_minutes: 15,  diet_type: "Gluten-Free"
+        get "/recipes.json"
+          result = JSON.parse(response.body)
+          expect(result[0]["title"]).to eq  "this is a title"
```
Comment (minachuong) [inline spec/requests/recipes_spec.rb:15]: there are a lot of other properties on the result you can assert on

File: spec/views/recipes/show.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+RSpec.describe "recipes/show", type: :view do
+  before(:each) do
+    @recipe = assign(:recipe, Recipe.create!())
+  end
+
+  it "renders attributes in <p>" do
+    render
+  end
+end
```
Comment (minachuong) [inline spec/views/recipes/show.html.erb_spec.rb:11]: if there's no show.html.erb files for recipes...is this file needed?

File: spec/views/recipes/new.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    assign(:recipe, Recipe.new())
+  end
+
+  it "renders new recipe form" do
+    render
+
+    assert_select "form[action=?][method=?]", recipes_path, "post" do
+    end
+  end
+end
```
Comment (minachuong) [inline spec/views/recipes/new.html.erb_spec.rb:14]: is this file needed?

File: spec/views/recipes/index.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    assign(:recipes, [
+      Recipe.create!(),
+      Recipe.create!()
+    ])
+  end
+
+  it "renders a list of recipes" do
+    render
+  end
+end
```
Comment (minachuong) [inline spec/views/recipes/index.html.erb_spec.rb:14]: is this file needed?

File: spec/views/recipes/edit.html.erb_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    @recipe = assign(:recipe, Recipe.create!())
+  end
+
+  it "renders the edit recipe form" do
+    render
+
+    assert_select "form[action=?][method=?]", recipe_path(@recipe), "post" do
+    end
+  end
+end
```
Comment (minachuong) [inline spec/views/recipes/edit.html.erb_spec.rb:14]: is this file needed?

**Review by minachuong — APPROVED** [review]
Please see comments

File: spec/views/recipes/show.html.erb_spec.rb (hunk 2 above)
Comment (csullivan710) [reply spec/views/recipes/show.html.erb_spec.rb:11]: I think we just missed deleting all these tbh. We dont need them, they were just auto created when we did our scaffolding
